pkgs/applications/networking/remote/citrix-workspace/sources.nix

```diff
@@ -87,6 +87,17 @@ let
       x86hash   = "f2dc1fd64e5314b62ba87f384958c2bbd48b06b55bed10345cddb05fdc8cffa1";
       x64suffix = "16";
       x86suffix = "16";
+      homepage  = "https://www.citrix.com/downloads/workspace-app/linux/workspace-app-for-linux/workspace-app-for-linux-2205.html";
```

Comment:
404. Correct URL is https://www.citrix.com/downloads/workspace-app/legacy-workspace-app-for-linux/workspace-app-for-linux-latest2.html